Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/cart/Cart.kt

```diff
@@ -150,12 +161,117 @@ private fun CartContent(
+                                            clip = true
+                                        ),
+                                    color = JetsnackTheme.colors.error
+
+                                ) {
+                                }
+                            }
+                            Row(
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .requiredHeight(132.dp)
+                                    .padding(end = paddingIcon),
+                                horizontalArrangement = Arrangement.End,
+                                verticalAlignment = Alignment.CenterVertically
+                            ) {
+                                if (offsetX < -40.dp && offsetX > -150.dp) {
+                                    val iconAlpha = if (offsetX < -100.dp) 0.5f else 1f
+                                    Icon(
+                                        imageVector = Icons.Filled.DeleteForever,
+                                        modifier = Modifier
+                                            .width(16.dp)
+                                            .height(16.dp)
+                                            .alpha(iconAlpha),
+                                        tint = JetsnackTheme.colors.uiBackground,
+                                        contentDescription = null,
+                                    )
+                                }
+                            }
+                            // Ensures that text moves to the left as the offset grows
+                            val paddingText = if (offsetX < 0.dp)(offsetX / -5) else 0.dp
+                            Row(
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .fillMaxHeight()
+                                    .padding(end = paddingText),
+                                horizontalArrangement = Arrangement.End,
+                                verticalAlignment = Alignment.CenterVertically
+                            ) {
+                                val textAlpha = if (offsetX > -140.dp) 0.5f else 1f
+                                if (offsetX < -120.dp) {
+                                    Text(
+                                        text = stringResource(id = R.string.remove_item),
+                                        style = MaterialTheme.typography.subtitle1,
+                                        color = JetsnackTheme.colors.uiBackground,
+                                        textAlign = TextAlign.Center,
+                                        modifier = Modifier
+                                            .alpha(textAlpha)
+                                    )
+                                }
+                            }
+                        }
+                    }
+                },
+                content = {
```

Comment:
Trailing lambda.

Reply:
Move the content parameter in SwipeDismissItem to the end so that content can be expressed with a trailing lambda (as "content" is implicit) in that pattern.

Reply:
@angeles-bilbao6

Reply:
Done!